SkyDrive/SkyDrive/Controllers/EventController.cs

```diff
@@ -38,6 +41,8 @@ public async Task<EventViewModel> GetEventById(int id)
         [HttpPost]
         public async Task<EventViewModel> CreateEvent(EventViewModel eventViewModel)
```

Comment:
Create new PR and add CancelationToken for all async endpoints

Reply:
OK. I'm going to do this ASAP